collections/_release_4_5/entry-platforms-linux-native-wayland-sub-window-support.md

```diff
@@ -0,0 +1,18 @@
+---
+type: entry
+section: platforms
+subsection: linux
+rank: 0
+importance: 2
+anchor: native-wayland-sub-window-support
+title: Native Wayland sub-window support
+blockquote: One step closer to Wayland game embedding
+text: |
+  Game embedding is coming for Wayland. But the support for sub-windows was an important (and not a small) hurdle to cross in order to achieve this in the future.
```

Comment:
```suggestion
  Game embedding is coming to Wayland. But the support for sub-windows was an important (and not a small) hurdle to cross in order to be able to achieve this.
```

Reply:
I'm unclear - is game window embedding available in Wayland in 4.5?

Reply:
I wanted to tell people that sub-windows were a needed stepping stone towards game embedding. 

> Godot can now spawn new independent sub-windows when running on Wayland natively. Game window embedding in the editor is not yet available for Wayland, but will be coming soon.

I don't know if it tells the same story here.